docs/roundtripper.md

```diff
@@ -0,0 +1,128 @@
+## Tracing HTTP Outgoing Requests
+
+The tracer is able to collect data from the Go standard library for outgoing HTTP requests.
+That is, when one creates an [http.Client](https://pkg.go.dev/net/http@go1.21.3#Client) instance to make calls to HTTP servers.
+
+To achieve this data collection we provide a [RoundTripper](https://pkg.go.dev/net/http@go1.21.3#RoundTripper) wrapper to be used as the http.Client Transport.
+Additionally, as HTTP outgoing requests are exit spans, the HTTP request must be attached to a context containing a entry span.
+
+### Usage
+
+Let's assume the following code snippet as an example to be traced:
+
+```go
+client := &http.Client{}
+
+req, err := http.NewRequest(http.MethodGet, "https://www.instana.com", nil)
+
+_, err = client.Do(req)
+
+```
+
+The first thing we need is to add the collector to the project:
+
+```go
+col := instana.InitCollector(&instana.Options{
+  Service: "my-http-client",
+})
+```
+
+Then, we need to wrap the current Transport (if any) with `instana.RoundTripper`.
+If no Transport is provided, simply pass `nil` as the second argument.
+The `instana.RoundTripper` wrapper will intercept relevant information from the HTTP request, such as URL, methods and so on. It will then collect them to later be sent to the Agent.
```

Comment:
```suggestion
The `instana.RoundTripper` wrapper will intercept the relevant information from the HTTP request, such as URL, methods and so on. It will then collect them and send to the Agent periodically.
```